Issues 50, 55, 58, 60, 65 and 66

sekoukeita opened the pull request:
## Fixed issues 50, 55, 58, 60, 65 and 66.
- Moved the geoplatform drop down into menu dialog (issue #50 ). 
- Inside the menu, converted the drop down into a menu item that takes user's geoplatforms links as submenus for that menu item and allowed the icon in the link to also open the AGOL item in a new window (issue #66).
- Moved the 'Visible Columns', 'Publish To Geoplatform' and 'Export CSV' buttons from the tabs and put them right above (issue #55).
- Replaced the multi-select projects by an input that searches projects by Name, Site#, Site State, NPL 
Status, Description, EPA Region# or EPA Contact then select them from an autocomplete and add them on chips (issue #65).
- Fixed the bug on switching between tabs and added functionality to load projects data on adding and removing projects to the chips. The 'Go' button, probably no longer needed, has been commented out (issue #60)
-  Created a function in the frontend to generate the geojson instead of the backend (issue #58).

tbock commented:
@sekoukeita move the func.py to the scribe_models directory root and move the model you createed to a new models.py.  I think there is a missing migration file too.

sekoukeita commented:
@tbock The migrations where ignored. I added the missing one.
I moved the func.py into the scribe_models root folder.
I have also created project_explorer_models.py which contains the new model code. Do you want that model inside  scribe_models or in scribe_models/models?

tbock commented:
Put it in scribe_models/models @sekoukeita
